### ISSUE TITLE
Keep getting "[lncli] rpc error: code = Unknown desc = peer disconnected" when attempting to open channels with certain nodes

### ISSUE THREAD

**Issue by Z-and** [lightningnetwork/lnd#2930]
I've encountered this issue with about half the nodes I tried to open channels with. I can connect as peer without an issue but whenever I open a channel it stalls then eventually gives me the following output.

`[lncli] rpc error: code = Unknown desc = peer disconnected`

Below is my node status, everything looks to be okay. Not sure why this keeps happening.

```
    "identity_pubkey": "032f0d2d9f9b14b3a40488a1f2f255ecfd73b2272ae6d8399cbf067b0b55913281",
    "alias": "Zand",
    "num_pending_channels": 0,
    "num_active_channels": 5,
    "num_peers": 8,
    "block_height": 571088,
    "block_hash": "0000000000000000001d96002b130121b32b5b1d91bedbd637ae9900f1d9ce76",
    "synced_to_chain": true,
    "testnet": false,
    "chains": [
        "bitcoin"
    ],
    "uris": [
        "032f0d2d9f9b14b3a40488a1f2f255ecfd73b2272ae6d8399cbf067b0b55913281@173.52.56.208:9735"
    ],
    "best_header_timestamp": "1554931014",
    "version": "0.5.2-beta commit=v0.5.2-beta",
    "num_inactive_channels": 1
```

**Comment by Roasbeef** [comment]
Check your logs, the peer may be rejecting your channels if they're too small, or it already has enough channels.

**Comment by Z-and** [comment]
> Check your logs, the peer may be rejecting your channels if they're too small, or it already has enough channels.

Why were you so quick to close out this issue? I checked the logs and they are not showing anything. Neither of your suggestions apply to this particular scenario. One of the nodes I was trying to open a channel with belongs to a friends who doesn't have any channel restrictions

**Comment by Roasbeef** [comment]
`lnd` has a set of default channel restrictions. If you're connecting to other implementations, then they also have some restrictions as well. In your logs, look for `FNDG` and `PEER`. These will show why the funding fails.

**Comment by cfromknecht** [comment]
@Z-and what version of LND is your friend running? good chance this could a networking issue that should be resolved if both peers are on 0.6

**Comment by Z-and** [comment]
> `lnd` has a set of default channel restrictions. If you're connecting to other implementations, then they also have some restrictions as well. In your logs, look for `FNDG` and `PEER`. These will show why the funding fails.

I know that but in this case neither of those default restrictions apply. I was attempting to open a channel with .1 BTC capacity and my friend only has 12 channels open right now.

> @Z-and what version of LND is your friend running? good chance this could a networking issue that should be resolved if both peers are on 0.6

We are both running version 0.5.2

**Comment by cfromknecht** [comment]
> We are both running version 0.5.2

It's very likely the underlying issue is resolved by #2690, which will be in 0.6. If the issue stills persists after both sides are on 0.6, it might make sense to investigate further
